tests/cases/conformance/es2022/arbitraryModuleNamespaceIdentifiers/arbitraryModuleNamespaceIdentifiers_system.ts

```diff
@@ -0,0 +1,33 @@
+//@module: System
```

Comment:
You don't need individual tests for each module type. You can just use comma-separated variations in the twoslash comment and we'll run the test once for each variation:

For all possible values:
```js
// @module: *
```

For specific values:
```js
// @module: amd,commonjs,system,umd,es2015
```

For all values excluding specific values:
```js
// @module: -esnext,-node16
```